corehq/apps/locations/tasks.py

```diff
@@ -25,7 +25,7 @@
 
 
 @serial_task("{location_type.domain}-{location_type.pk}",
-             default_retry_delay=30, max_retries=3)
+             default_retry_delay=30, max_retries=3, serializer='pickle')
 def sync_administrative_status(location_type):
```

Comment:
I'm pretty sure `location_type` is a `LocationType` object (django model), so this interface will have to be rewritten more substantially.